tests/topotests/lib/common_config.py

```diff
@@ -2281,7 +2281,7 @@ def configure_vxlan(tgen, input_dict):
                             cmd = "{} local {}".format(cmd, local_addr)
 
                         if learning == "no":
-                            cmd = "{} {} learning".format(cmd, learning)
+                            cmd = "{} nolearning".format(cmd, learning)
```

Comment:
got a format entry with no replacement? was this meant to be 'learning' vs 'nolearning', or does it always have to be 'nolearning' ?

Reply:
I believe always nolearning..